packages/libs/client/src/interfaces/type-utilities.ts

```diff
@@ -39,3 +39,15 @@ export type ExtractCapabilityType<TCommand> = TCommand extends {
       : IGeneralCapability
     : IGeneralCapability
   : IGeneralCapability;
+
+export type Prettify<T extends Record<string, unknown>> = {
```

Comment:
Do/should we have tests for these utility types?

Reply:
good idea we can write tests for them